### ISSUE TITLE
OS Workflow should not request outputs, especially timestep, by default

### ISSUE THREAD

**Issue by shorowit** [NREL/OpenStudio#5188]
Enhancement Request
--------------
By default, an OpenStudio workflow will request a variety of [monthly](https://github.com/NREL/OpenStudio/blob/87db4fb768f0dca378943a372b369c8c85a32670/src/workflow/RunPreProcessMonthlyReports.hpp#L9) and, worse, [timestep/daily](https://github.com/NREL/OpenStudio/blob/87db4fb768f0dca378943a372b369c8c85a32670/src/workflow/RunPreProcessMonthlyReports.hpp#L234) outputs. For users that care about runtime speed or file size, this is problematic. Why does OpenStudio assume everyone wants these outputs? If certain reports or workflows or applications need these outputs, they can easily request them.

(Technically these can be skipped by using the `skip_energyplus_preprocess` option, but that's not easy to know and many end users may not have access to this option with how they are running simulations. And this option might do more than just skip these outputs, I'm not sure.)

Issue originally raised on [Unmet Hours](https://unmethours.com/question/98880/disabling-default-outputmeter-in-openstudio/?answer=98917).

**Comment by jmarrec** [comment]
I don't really know, this was already in workflow-gem, I just ported it.

My money is on the fact that this was added for the openstudio_results measure (and maybe the Output:Table:Monthly so that the E+ html output is more useful too).

@DavidGoldwasser could we ensure these are ported to the openstudio_results measure instead of always added by the Workflow?

**Comment by DavidGoldwasser** [comment]
That makes sense. I think when that code was added pre-dated reporting measures and it was more hard coded in OS App. We should be able to add what we want to the EnergyPlus output request section of OpenStudio results measure. Will have to check if updates also needed for calibration measures and measures in URBANopt.

**Comment by DavidGoldwasser** [comment]
Just looking back that this, we can remove any outputs that are added, but document which ones are removed and I can add those to output requests of OpenStudio Results measure. People may get in trouble if they update to OpenStudio 3.10.0 using an old version of OpenStudio Results, but all they have to do is update it. @MatthewSteen I assume this won't be problematic for Autodesk.

**Comment by MatthewSteen** [comment]
I agree that the default behavior should be _only_ yearly and monthly outputs for high level results reporting.

@DavidGoldwasser yes, we can adjust for Autodesk if needed. I don't think they use timestep frequency outputs. 

I think I recall that the OpenStudio Results measure might use timestep-level outputs for some of its reporting, but you would know better. Again, I think the default behavior for the measure should also be _only_ yearly and monthly.
